app/workers/commit_monitor_handlers/commit_range/gemfile_checker.rb

```diff
@@ -0,0 +1,101 @@
+module CommitMonitorHandlers
+  module CommitRange
+    class GemfileChecker
+      include Sidekiq::Worker
+      sidekiq_options :queue => :miq_bot
+
+      LABEL_NAME = "gem changes".freeze
+
+      def self.handled_branch_modes
+        [:pr]
+      end
+
+      attr_reader :branch, :commits, :github, :pr
+
+      def perform(branch_id, _new_commits)
+        @branch = CommitMonitorBranch.where(:id => branch_id).first
+        
+        if @branch.nil?
+          logger.info("Branch #{branch_id} no longer exists.  Skipping.")
```

Comment:
Can you review and normalize the log message prefix for the checker?  This wasn't clear before.   I believe sidekiq will prefix all of your messages in the worker with the `GemfileChecker` value... can you grep your logs to see if it's consistent throughout this worker?  I'm not sure if the `self.class` used below in the log lines will be redundant.